packages/cli/src/install/install.ts

```diff
@@ -14,14 +14,17 @@
 
 import * as bower from 'bower';
 import {read as readBowerJson} from 'bower-json';
+import * as child_process from 'child_process';
 import * as path from 'path';
 import * as logging from 'plylog';
+import * as util from 'util';
 
 import defaultBowerConfig = require('bower/lib/config');
 import BowerLogger = require('bower-logger');
 import StandardRenderer = require('bower/lib/renderers/StandardRenderer');
 import BowerProject = require('bower/lib/core/Project');
 
+const exec = util.promisify(child_process.exec);
```

Comment:
This is sadly not available in node 6, we just aren't running node 6 integration tests currently (see #136)